news/2021-04-11-osu-beatmapping-world-championship-2021-call-for-judges.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: "osu! Beatmapping World Championship 2021: Call for Judges"
+date: 2021-04-11 14:00:00 +0000
+---
+
+With the Spring season comes the next iteration of osu! Beatmapping World Championship! If you're excited and eager to help us out, here's your chance: We are looking for judges!
+
+![](/wiki/shared/news/banners/obwc2021.png)
+
+For the third iteration of osu! Beatmapping World Championship, we have taken your feedbacks into account and implemented some significant changes in order to make this contest enjoyable for everyone. To help us with judging, we are looking for mappers, modders, and this time even players! Looking for where to apply? Just keep on reading, and you'll find out!
+
+## osu! Beatmapping World Championship 2021
+
+For more information, check out the [wiki page](/wiki/Contests/oBWC/3) and the [official website](https://obwc.net/).
```

Comment:
```suggestion
For more information, check out the [wiki page](/wiki/Contests/oBWC/3) and the [official website](https://obwc.net/).
```
official website needs alt text; i can't seem to access the page for some reason tho (connection closed)